src/downshift.js

```diff
@@ -621,7 +630,7 @@ class Downshift extends Component {
     this.internalSetState({
       type: Downshift.stateChangeTypes.changeInput,
       isOpen: true,
-      inputValue: event.target.value,
+      inputValue: isReactNative() ? event : event.target.value,
```

Comment:
This seems a bit odd, since the `onChangeText` prop on React Native returns just the text as a string, not an object. Is there a better way to go about this?